### PR TITLE
Fix/travis parallel jobs

### PR DESCRIPTION
**Pull request by KickSeason**
* use travis parallel jobs to shorten test time.

### CODE REVIEW

**Comment by shargon** [comment]
codecov will work with both TEST_SUIT?

File: .travis.yml [hunk 1]
```diff
@@ -10,16 +10,26 @@ osx_image: xcode9.1
...
   - cd neo.UnitTests
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ulimit -n 2048; fi
+script: |
+  if [[ "$TEST_SUITE" == cultures ]]; then 
+      dotnet restore
+      dotnet test
+  else
+      dotnet restore
+      find * -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
+      dotnet test --filter FullyQualifiedName!=Neo.UnitTests.UT_Culture.All_Tests_Cultures /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
```
Comment (vncoelho) [inline .travis.yml]: `-v n` is needed?

How about export also as json? 
` \"json,opencover\"`

**Comment by KickSeason** [comment]
@shargon Yes, it will.

File: .travis.yml (hunk 1 above)
Comment (KickSeason) [reply .travis.yml]: what json format reports are used for?

Comment (KickSeason) [reply .travis.yml]: *  -v n. We can see the tests passed and time it takes. Good idea.
* What json format report used for?

**Comment by eryeer** [comment]
"Run `All_Tests_Cultures`+ Compute code coverage" together cost a bunch of time.  In fact after we continue to add UT code to `Wallets` module, travis has timed out(>50min). We need to exclude `All_Tests_Culturesmethod ` from the code coverage computation.

**Comment by shargon** [comment]
Seems good to me

**Comment by Tommo-L** [comment]
Hi @vncoelho, could we merge now? Wallets uint tests are blocked by this PR. 😂

**Pull request merged**